aten/src/ATen/core/Tensor.h

```diff
@@ -441,12 +441,10 @@ struct AT_API Tensor {
   Tensor & atan_();
   Tensor baddbmm(const Tensor & batch1, const Tensor & batch2, Scalar beta=1, Scalar alpha=1) const;
   Tensor & baddbmm_(const Tensor & batch1, const Tensor & batch2, Scalar beta=1, Scalar alpha=1);
-  Tensor bernoulli(const Tensor & p, Generator * generator=nullptr) const;
```

Comment:
Err, was removing this intentional?

Reply:
Yep. It didn't quite make any sense and not used. 
What it used to do: sample `Bernoulli` according to each value in `p`. It doesn't have anything to do with `self`!!!
So it makes much more sense to just call `torch.bernoulli(p)` instead.